src/components/Task.jsx

```diff
@@ -1,21 +1,39 @@
-import TaskCard from 'components/ui/TaskCard';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TaskFooter from 'components/ui/TaskFooter';
 
-export default function Task({ task }) {
-  const { taskDetails, createdAt } = task;
-  return <TaskCard details={taskDetails} createdAt={createdAt} />;
+function Task({ task }) {
+  const { id, taskDetails, createdAt, isCompleted } = task;
+
+  const classes = classNames({
```

Comment:
```suggestion
  const TaskDetailsMainDiv= classNames({
```

Reply:
@fza-wtag IMHO, wouldn't it be better to use camelCase instead of PascalCase?
like this:
```suggestion
  const taskDetailsMainDiv = classNames({
```

Reply:
@fza-wtag bhai, @iud-wtag bhai, I have renamed the variable according to the classnames [doc](https://www.npmjs.com/package/classnames). thanks.